### DISCUSSION TITLE
Mouse cursor on mobile is too big and screen can't be rotated manually

### DISCUSSION THREAD

**Discussion by kidzgy** [rustdesk/rustdesk#8477] [General]
Version: rustdesk-1.2.6-armv7-signed.apk

![7a15792b8c7e1dbfbef616ef1c4905d](https://github.com/rustdesk/rustdesk/assets/6967810/b44a866f-e3ab-4fa2-ba8d-f637b2de8ced)

As shown in the picture, this is a screenshot from my cell phone:
(1) On a cell phone screen, the cursor displays a very large pointer, making it difficult to locate it precisely.
(2) It cannot be rotated manually. My phone configuration is locked to portrait orientation by default, but I want the app to not be controlled by the system settings, I want the orientation to rotate depending on the rotation of the phone, and I want to add a manual rotate button so that I can control the screen orientation accurately and easily.

Translated with DeepL.com (free version)

**Comment by xlionjuan** [comment]
Did you know Android have this?

![Screenshot_20240627-171851.png](https://github.com/rustdesk/rustdesk/assets/25243894/7e5cbe85-cda2-4b8e-a154-057f94d00697)

**Reply by kidzgy** [reply]
> Did you know Android have this?
> 
> ![Screenshot_20240627-171851.png](https://private-user-images.githubusercontent.com/25243894/343711014-7e5cbe85-cda2-4b8e-a154-057f94d00697.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0ODAzMDMsIm5iZiI6MTcxOTQ4MDAwMywicGF0aCI6Ii8yNTI0Mzg5NC8zNDM3MTEwMTQtN2U1Y2JlODUtY2RhMi00YjhlLWExNTQtMDU3Zjk0ZDAwNjk3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI3VDA5MjAwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThjY2U0ZTZiZDZhYTdmYzJmNjI4MTExZmEwNjU2YWQzYmI2ODczNWEzYWRhYjI4NzU3YmRlNjZlNmFhNmM5ZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OdjcoT45DowOthipy1XX3pmXfR6QbkUBT-AENj6JHNM)

My android version is 10, MIUI 12, and there is no such rotate button, I would like the rotate button to be built into the app.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/8478

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/16144
